execute/entities.go

```diff
@@ -48,3 +51,17 @@ const (
 	// Skipped indicates a client' TestCase did not run
 	Skipped
 )
+
+var StatusNames = map[Status]string{
+	Success: "success",
+	Failed:  "failed",
+	Skipped: "skipped",
+}
```

Comment:
Tried it. JSON marshal doesn’t pick it up. I’ll push a switch/case variant of MarhsalText and use that instead of the table, but the String overload doesn’t fly.